### ISSUE TITLE
Voice cloning paused due to a requested directory that doesn't exist.

### ISSUE THREAD

**Issue by TROLL-PIRATE** [JarodMica/ai-voice-cloning#44]
Hello

I'm not really good at code. But i've been spending hours trying to understand why i can't voice clone when using RVC.
I made sure that the compute and Cuda were compatible.

I believe it's to do with a directory it's requesting.

2024-02-12 03:40:24 | INFO | httpx | HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
[Training] [2024-02-12T03:40:37.154493] 24-02-12 03:40:37.154 - INFO: Loading model for [./models/tortoise/autoregressive.pth]
2024-02-12 03:40:37 | INFO | httpx | HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
[Training] [2024-02-12T03:40:38.940063] 24-02-12 03:40:38.930 - INFO: Start training from epoch: 0, iter: 0
[Training] [2024-02-12T03:40:41.625444] [2024-02-12 03:40:41,625] torch.distributed.elastic.multiprocessing.redirects: [WARNING] NOTE: Redirects are currently not supported in Windows or MacOs.
[Training] [2024-02-12T03:40:44.176698] [2024-02-12 03:40:44,176] torch.distributed.elastic.multiprocessing.redirects: [WARNING] NOTE: Redirects are currently not supported in Windows or MacOs.
[Training] [2024-02-12T03:40:46.535774] D:\AI_2023\ai-voice-cloning\runtime\lib\site-packages\torch\optim\lr_scheduler.py:136: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
[Training] [2024-02-12T03:40:46.535774]   warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
[Training] [2024-02-12T03:40:47.640751] D:\AI_2023\ai-voice-cloning\runtime\lib\site-packages\torch\utils\checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
[Training] [2024-02-12T03:40:47.640751]   warnings.warn(

-----------------------------This part here-----------------------------------------------------------------------------------

**[Training] [2024-02-12T03:40:54.738021] Error no kernel image is available for execution on the device at line 167 in file D:\ai\tool\bitsandbytes\csrc\ops.cu**

--------------------------------------------------------------------------------------------------------------------------------

I'm not sure i understand what to do here.

The result leads to the cloning process being paused.

[Training] [2024-02-12T03:40:55.127367]
[Training] [2024-02-12T03:40:55.127367] D:\AI_2023\ai-voice-cloning>pause

Thanks for great work.

**Comment by Zaffaci** [comment]
Found a solution for "Error no kernel image is available for execution on the device at line 167 in file D:\ai\tool\bitsandbytes\csrc\ops.cu" seems like that is an error occured only between bitsandbytes and Nvidia 1xxx gpu series

Be sure that you have cuda 11.6 [(https://developer.nvidia.com/cuda-11-6-0-download-archive)](https://developer.nvidia.com/cuda-11-6-0-download-archive)
Download the latest version of bitsandbytes [(https://github.com/TimDettmers/bitsandbytes/archive/refs/tags/0.42.0.zip)](https://github.com/TimDettmers/bitsandbytes/archive/refs/tags/0.42.0.zip) and copy everything from the folder "bitsandbytes-0.42.0\bitsandbytes" to "ai-voice-cloning\runtime\Lib\site-packages\bitsandbytes"

**Comment by freeload101** [comment]
> Found a solution for "Error no kernel image is available for execution on the device at line 167 in file D:\ai\tool\bitsandbytes\csrc\ops.cu" seems like that is an error occured only between bitsandbytes and Nvidia 1xxx gpu series
> 
> Be sure that you have cuda 11.6 [(https://developer.nvidia.com/cuda-11-6-0-download-archive)](https://developer.nvidia.com/cuda-11-6-0-download-archive) Download the latest version of bitsandbytes [(https://github.com/TimDettmers/bitsandbytes/archive/refs/tags/0.42.0.zip)](https://github.com/TimDettmers/bitsandbytes/archive/refs/tags/0.42.0.zip) and copy everything from the folder "bitsandbytes-0.42.0\bitsandbytes" to "ai-voice-cloning\runtime\Lib\site-packages\bitsandbytes"

OMG I tried like 10 differnt things on my crap 8GIG GPU 1070 and this worked! THANK YOU SO MUCH !
